riotfile.py

```diff
@@ -380,25 +366,38 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     pkgs={
                         "gevent": "~=1.3.0",
                         "greenlet": "~=1.0",
+                        "requests": "==2.20.0",
+                        "opensearch-py": "==1.0.0",
```

Comment:
should we have comments for any of the places where we have exceptions?

it is unclear to me why we need to pin these here instead of relying on `latest` + `pip-compile` results?